### ISSUE TITLE
ReplyKeyboardMarkup group chats

### ISSUE THREAD

**Issue by Qnetis** [python-telegram-bot/python-telegram-bot#3672]
### Issue I am facing

Hi. Custom keyboard is attached to the response of the send message.
Using ReplyKeyboardMarkup
When you directly contact the bot , the message is not tied to the buttons . When in a group chat , all the time the message hangs tied to the buttons . 
Is there a way to remove the binding in group chat?
Example.
group chat:
![image](https://user-images.githubusercontent.com/23212415/233616372-56b84d49-f1f2-496f-a627-09cb0dfb7cf2.png)
private messages:
![image](https://user-images.githubusercontent.com/23212415/233616392-da9091f6-4e01-4424-9d51-833ae1727e0e.png)

### Related part of your code

```python
reply_keyboard = [["Boy", "Girl", "Other"]]
    keyboard = ReplyKeyboardMarkup(reply_keyboard, one_time_keyboard=True, resize_keyboard=True,force_reply=False)
    await context.bot.send_message(chat_id=update.effective_chat.id,
                                   text="Are you a boy or a girl?",
                                   reply_markup=keyboard,
                                   )
```

### Operating System

windows 10

### Version of Python, python-telegram-bot & dependencies

```shell
python-telegram-bot 20.0a2
Bot API 6.1
Python 3.10.8 (tags/v3.10.8:aaaf517, Oct 11 2022, 16:50:30) [MSC v.1933 64 bit (AMD64)]
```

**Comment by harshil21** [comment]
Hi,

This is directly related to the application of the `force_reply` field of `ReplyKeyboardMarkup`. If you keep it as `False`, then that will not happen. If you previously set it to `True` for testing purposes, but did not actually send a message, then a "ghost" force reply is present. To remove that, either delete the message it is referencing, or just send a message.

P.S. You should update your PTB version since alphas were not stable (`pip install -U python-telegram-bot`)

**Comment by Qnetis** [comment]
thanks for the reply.
force_reply was added recently as an experiment, it worked the same way without it. Now I have removed it and the result has not changed.
I tried to delete the message referenced, the buttons are deleted along with it.

    msg_id=await context.bot.send_message(chat_id=update.effective_chat.id,
                                   text="Are you a boy or a girl?",
                                   reply_markup=keyboard,
                                   )

    await context.bot.delete_message(chat_id=update.effective_chat.id, message_id=msg_id.message_id)
 P.S. updated the version, thanks)

**Comment by harshil21** [comment]
It can differ from client to client, or it might just be a bug in the client. 

btw if you don't directly reply to the message from the bot in a group chat, and your bot has privacy mode enabled, it will not receive the reply. See https://github.com/python-telegram-bot/python-telegram-bot/wiki/Frequently-Asked-Questions#what-messages-can-my-bot-see

**Comment by Qnetis** [comment]
Thanks for the answer, but that's not what I'm looking for. It is not possible to send ReplyKeyboardMarkup without text.
I don 't use Force Reply .
Is it possible to send custom keyboard in group chats without a reply message ?
I've tried everything and it doesn't work

**Comment by harshil21** [comment]
> Is it possible to send custom keyboard in group chats without a reply message?

No, you have to send a message if you want to send a keyboard

**Comment by harshil21** [comment]
Okay I just tested it myself in a group. It is a Telegram Desktop client bug.

On desktop: Using the `ReplyKeyboardMarkup` shows both the keyboard and "reply" interface, but when you send the message, it doesn't actually "reply" to that message.

On Android: There is no "reply" interface, only the buttons are shown.

**Comment by harshil21** [comment]
tl;dr: this behaviour cannot be changed, it is not handled by PTB, but the Telegram clients itself
